web/src/components/AuthButton.tsx

```diff
@@ -9,7 +9,7 @@ import { useRouter } from "next/router";
 import { ReactElement, useContext } from "react";
 
 interface Props {
-  position: "HERO" | "NAVBAR";
+  position: "NAVBAR";
```

Comment:
this prop does not need to exist at all if it only ever has one possible value. It should just BE the value, it doesn't need to come from a prop anymore